docs/client_reference.rst

```diff
@@ -1594,6 +1594,31 @@ manually.
          The method is converted into :term:`coroutine`,
          *compress* parameter added.
 
+   .. method:: send_frame(message, opcode, compress=None)
+      :async:
+
+      Send a :const:`~aiohttp.WSMsgType` message *message* to peer.
+
+      This method is low-level and should be used with caution as it
+      only accepts UTF-8 encoded bytes for *message*.
```

Comment:
Isn't this untrue if opcode is BINARY? Maybe more accurate to say that the bytes must conform to the correct message type.

Reply:
Yes, thats better